<ui version="4.0" >
 <class>timerui</class>
 <widget class="QWidget" name="timerui" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>254</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Set the time between 2 letters.</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <property name="horizontalSpacing" >
        <number>4</number>
       </property>
       <property name="verticalSpacing" >
        <number>4</number>
       </property>
       <item row="2" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>219</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="QSlider" name="kcfg_KidTimer" >
         <property name="toolTip" >
          <string>Set the timer (in tenths of seconds)</string>
         </property>
         <property name="whatsThis" >
          <string>You can set the time between two letters in Kid mode. Default is 4 tenths of seconds but younger children might need longer time.</string>
         </property>
         <property name="maximum" >
          <number>10</number>
         </property>
         <property name="pageStep" >
          <number>1</number>
         </property>
         <property name="value" >
          <number>4</number>
         </property>
         <property name="sliderPosition" >
          <number>4</number>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition" >
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval" >
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Kid Mode&lt;/p>&lt;/body>&lt;/html></string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" >
        <widget class="QLabel" name="timeKidLabel" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <property name="horizontalSpacing" >
        <number>4</number>
       </property>
       <property name="verticalSpacing" >
        <number>4</number>
       </property>
       <item row="2" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>219</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="QSlider" name="kcfg_GrownTimer" >
         <property name="toolTip" >
          <string>Set the timer (in tenths of seconds)</string>
         </property>
         <property name="whatsThis" >
          <string>You can set the time between two letters in Grown-up mode. Default is 2 tenths of seconds.</string>
         </property>
         <property name="maximum" >
          <number>10</number>
         </property>
         <property name="pageStep" >
          <number>1</number>
         </property>
         <property name="value" >
          <number>2</number>
         </property>
         <property name="sliderPosition" >
          <number>2</number>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition" >
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval" >
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Grown-up Mode&lt;/p>&lt;/body>&lt;/html></string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" >
        <widget class="QLabel" name="timeGrownLabel" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
